packages/suite/src/hooks/wallet/useSendFormImport.ts

```diff
@@ -32,6 +27,29 @@ export const useSendFormImport = ({ network, tokens, localCurrencyOption, fiatRa
         const result = await dispatch(importSendFormRequestThunk()).unwrap();
         if (!result || result.length < 1) return; // cancelled
 
+        let currentRates = [];
+        const currencies = result.map(it => it.currency.toLowerCase());
+        const uniqueCurrencies = [...new Set(currencies)];
+
+        for (const currency of uniqueCurrencies) {
+            const updateFiatRatesResult = await dispatch(
+                updateFiatRatesThunk({
+                    ticker: {
+                        symbol: network.symbol as NetworkSymbol,
+                    },
+                    localCurrency: currency as FiatCurrencyCode,
+                    rateType: 'current',
+                    fetchAttemptTimestamp: Date.now() as Timestamp,
+                }),
+            );
+
+            if (updateFiatRatesResult.meta.requestStatus === 'fulfilled') {
+                const rate = updateFiatRatesResult.payload as fiatRatesResult;
+
+                currentRates.push({ currency, rate: rate.rate });
+            }
+        }
+
```

Comment:
Not ideal approach but accessing it with selector seems to not be possible since it doesn't update itself (state is updated but value from selector not)

Reply:
Why it is not updated?

Reply:
Can you just fetch what is not yet available in the store?

Reply:
Sure, will reuse `selectShouldUpdateFiatRate` in `updateFiatRatesThunk`

Reply:
a1718ae7de5e8b020f8b1c9b763eee8039cc6bd1